.github/workflows/ci.yaml

```diff
@@ -14,7 +14,7 @@ permissions:
 
 jobs:
   lint-shell:
-    runs-on: [self-hosted, Linux, small]
+    runs-on: ubuntu-22.04
```

Comment:
probably doesn't matter for a while, thoughts on `ubuntu-latest`?

Reply:
For this stuff, either way should be fine.